crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -960,6 +968,236 @@ pub enum SuiInputObjectKind {
     },
 }
 
+/// A series of commands where the results of one command can be used in future
```

Comment:
@patrickkuo sorry for grouping this in this PR, but the changes felt at least separate enough (there are no other changes to this file other than for this addition).

Let me know though if you want this changed to a different PR!

Reply:
No problem!